repository.yaml

```diff
@@ -1 +1,2 @@
-repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
\ No newline at end of file
+repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
+digital-service: single-customer-account-frontend
```

Comment:
Why has this been added?

Reply:
This I believe is created as part of Jenkins build.

Reply:
Actioned